spec/integration_spec.rb

```diff
@@ -359,167 +355,6 @@
   end
 end
 
-describe 'Book' do
-  before do
-    Book.clear_index!(true)
-    Book.index(safe_index_uid('BookAuthor')).delete_all_documents
-    Book.index(safe_index_uid('Book')).delete_all_documents
-  end
-
-  it 'returns array of tasks on #ms_index!' do
-    moby_dick = Book.create! name: 'Moby Dick', author: 'Herman Melville', premium: false, released: true
-
-    tasks = moby_dick.ms_index!
-
-    expect(tasks).to contain_exactly(
-      a_hash_including('uid'),
-      a_hash_including('taskUid'),
-      a_hash_including('taskUid')
-    )
-  end
-
-  it 'indexes the book in 2 indexes of 3' do
-    steve_jobs = Book.create! name: 'Steve Jobs', author: 'Walter Isaacson', premium: true, released: true
-    results = Book.search('steve')
-    expect(results.size).to eq(1)
-    expect(results).to include(steve_jobs)
-
-    index_author = Book.index(safe_index_uid('BookAuthor'))
-    expect(index_author).not_to be_nil
-    results = index_author.search('steve')
-    expect(results['hits'].length).to eq(0)
-    results = index_author.search('walter')
-    expect(results['hits'].length).to eq(1)
-
-    # premium -> not part of the public index
-    index_book = Book.index(safe_index_uid('Book'))
-    expect(index_book).not_to be_nil
-    results = index_book.search('steve')
-    expect(results['hits'].length).to eq(0)
-  end
-
-  it 'sanitizes attributes' do
```

Comment:
Moved to `options_spec.rb` since it tests whether or not the `:sanitize` option is behaving correctly.